mediar-ai/screenpipe
# Screen time storyteller pipe

# [PR] Implement Screen Time Storyteller Pipe

## Description
This PR implements a new pipe called "Screen Time Storyteller" for Screenpipe. The pipe analyzes user's 24/7 screen recordings and generates personalized, humorous roasts about their digital habits. It aims to provide insights and motivation for improving productivity through AI-powered feedback.

Related issue: #365 
Fixes: #365
/claim #365 

## Type of change

## How to test
1. Checkout this branch
2. Install dependencies
3. Run Screenpipe with the new Screen Time Storyteller pipe enabled
4. Check the generated narrative summary for personalized insights and humor

## Screenshot
<img width="910" alt="Screenshot 2024-10-08 at 00 35 25" src="https://github.com/user-attachments/assets/294f26c5-2836-4538-b474-cfe1581e262a">


## Additional notes
A YouTube video demonstrating the Screen Time Storyteller pipe has been created and published at [https://youtu.be/XMVvkk0M7jQ](https://youtu.be/XMVvkk0M7jQ). The video showcases the functionality and DIY

No Reddit post was made; not a frequent Redditor so i don't have enough karma to post.
